spec/path-scanner-spec.coffee

```diff
@@ -136,6 +136,17 @@ describe "PathScanner", ->
         runs ->
           expect(paths).toContain path.join(rootPath, 'dir', 'file7_ignorable.rb')
 
+      it "local directory inclusions don't override global subdirectory exclusions", ->
```

Comment:
I agree, I was following the convention from the existing surrounding specs (line 129 & 150). I'm happy to reword those too.